tests/unit/conftest.py

```diff
@@ -1,28 +1,33 @@
-# pylint:disable=redefined-outer-name
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
 
 import re
+from collections.abc import Iterator
 from random import randint
-from typing import Any, Dict
+from typing import Any
+from unittest.mock import MagicMock
```

Comment:
For my learning, could you please comment what this is (used for)?

Reply:
MagicMock?  mocks is a really important tool for testing. Google "python mock with pytest" . we can also talk about it separately then

Reply:
Here a gentle intro https://changhsinlee.com/pytest-mock/